docs/sources/panels/field-configuration-options.md

```diff
@@ -105,37 +101,6 @@ This section explains all available field options. They are listed in alphabetic
 
 Most field options will not affect the visualization until you click outside of the field option box you are editing or press Enter.
 
-### Cell display mode
```

Comment:
Why was this content removed?

Reply:
it was moved to the table visualization, as these options are all table specific display options (custom field config options)